src/src.pro

```diff
@@ -129,6 +129,9 @@ macx {
     LIBS += -L/usr/local/lib
     INCLUDEPATH += /usr/local/include
 }
+windows {
+    DEFINES *= Q_COMPILER_INITIALIZER_LISTS
```

Comment:
on MSVC2013, there is a "initializer-list" problem, like this blog.
http://blog.csdn.net/a312024054/article/details/72784315